modules/benchmark/R/load_data.R

```diff
@@ -26,13 +27,27 @@ load_data <- function(data.path, format, start_year = NA, end_year = NA, site =
     fcn <- match.fun(fcn1)
   } else if (exists(fcn2)) {
     fcn <- match.fun(fcn2)
+  } else if (!exists(fcn1) & !exists(fcn2)) { 
+    #To Do: call to DAP to see if conversion to csv is possible
+    #Brown Dog API call through BDFiddle, requires username and password
+    #will be made into R package
+    install_github("yan130/bd.r")
+    
+    key   = get_key("https://bd-api.ncsa.illinois.edu",username,password) #generic
+    token = get_token("https://bd-api.ncsa.illinois.edu",key)
+    #output_path = "/fs/data3/jam2767/veg_test" where are we putting converted file?
+    convert_file(url = "https://bd-api.ncsa.illinois.edu", 
+                 input_filename = paste0(data.path, format$file_name), 
+                 output = "csv", output_path, token)
+    #not doing anything about mimetypes not convertible by BD right now
+    fcn <- match.fun(fcn2)
```

Comment:
In addition to updating the fcn to fcn2, you also need to update the data.path to the new file returned from BD. Otherwise, when you call fcn a few lines later you'll be passing it the original, unconverted, file.